modules/cloud-sql/main.tf

```diff
@@ -39,10 +46,9 @@ resource "google_sql_database_instance" "master" {
   database_version = var.engine
 
   settings {
-    tier                        = var.machine_type
-    activation_policy           = var.activation_policy
-    authorized_gae_applications = var.authorized_gae_applications
-    disk_autoresize             = var.disk_autoresize
+    tier              = var.machine_type
+    activation_policy = var.activation_policy
+    disk_autoresize   = var.disk_autoresize
```

Comment:
Are these Terraform 0.13 changes or new GCP provider changes? I'm guessing the latter?

Reply:
Product management changes actually! I was going to have to work around a bit of the Google App Engine stuff, so I'm actually planning to deprecate it! I assuming its not the direction most of our customers are going in and none of the Google maintained Open Source modules offer support for it either.